README.md

```diff
@@ -43,6 +43,55 @@ Optional parameters:
   - `az,type`
   - `id,publicIP,launchTime`
 
+### /ec2/all ###
+Search all ec2 instances.
+
+### /rds/all ###
+Search all RDS resources.
+
+### /dynamo ###
+
+Search tables in DynamoDB.
+
+Optional parameters:
+
+- `q` / `query`: OGNL expression used to filter. Example:
+  - `id > 100 && readCapacityUnits > 10`
+- `s` / `sort`: OGNL expression used as a ``Comparable`` to sort (default to no ordering). Example:  
+  - `s=itemCount`
+- `l` / `limit`: maximum number of records to return. Example:
+  - `l=10`
+- `f` / `field`: comma-separated list of fields to display(defaults to all). Examples:
+  - `f=tableName,tableStatus`
+  - `f=tableName`
+
+Sample response (the ``readCapacityUnits`` and ``writeCapacityUnits`` are pulled out of ``provisionedThroughput`` which allows users to sort with these attributes):
```

Comment:
I'll rename the "sample response" into "attributes" to show all the fields of the table.